flocker/apiclient/_client.py

```diff
@@ -25,14 +25,14 @@ class Dataset(PClass):
     A dataset in the configuration.
 
     :attr UUID primary: The node where the dataset should manifest.
-    :attr int maximum_size: Size of new dataset, in bytes.
+    :attr maximum_size: Size of new dataset in bytes (as ``int``) or ``None``.
```

Comment:
In `DatasetState`, the code has `:attr FilePath|None path:`. This "or-typing" could be applied to these changed definitions, or that definition could be changed to match this "prose-typed" definition, but we should be consistent within a module, at least.